src/libs/OptionsListUtils.ts

```diff
@@ -806,6 +808,9 @@ function createOption(
         result.isWaitingOnBankAccount = report.isWaitingOnBankAccount;
         result.policyID = report.policyID;
         result.isSelfDM = ReportUtils.isSelfDM(report);
+        result.isTripRoom = ReportUtils.isTripRoom(report);
+        result.policyName = ReportUtils.getPolicyName(report);
+        result.isSearchableViaParticipants = ReportUtils.isSearchableViaParticipants(report);
```

Comment:
NAB, but not a fan of this still

Reply:
I actually removed these, not used anymore!